azkaban-common/src/main/java/azkaban/jobtype/JobTypeManager.java

```diff
@@ -160,6 +163,39 @@ private void loadPluginJobTypes(final JobTypePluginSet plugins)
     plugins.setCommonPluginJobProps(commonPluginJobProps);
     plugins.setCommonPluginLoadProps(commonPluginLoadProps);
 
+    if (commonPluginJobProps
```

Comment:
The place where we have used is not the place where it can be overridden. It gets overridden when passed to the JobType class.
However, commonPluginLoadProps seems more appropriate in this case as these are indeed load-time props. Hence changed it to read from private properties.